### PR TITLE
Add dashboards from g8s grafana

### PR DESCRIPTION
**Pull request by QuentinBisson**
This PR

- adds private dashboard from g8s-grafana towards https://github.com/giantswarm/roadmap/issues/518

### Checklist

- [x] Update changelog in CHANGELOG.md in an end-user friendly language.

Tested on gauss

To merge and release with https://github.com/giantswarm/g8s-grafana/pull/384

### CODE REVIEW

File: helm/dashboards/dashboards/aws/public/aws-cluster-info.json [hunk 1]
```diff
@@ -1283,4 +1283,4 @@
   "title": "AWS Cluster Status",
   "uid": "ewKOQTmnk",
   "version": 1
-}
```
[the remark this answers is not in the document]
Comment (QuentinBisson) [reply helm/dashboards/dashboards/aws/public/aws-cluster-info.json:1286]: Probably a typo from me

File: helm/dashboards/dashboards/kvm/kvm-machine-usage.json [hunk 2]
```diff
@@ -1,3046 +0,0 @@
-{
```
[the remark this answers is not in the document]
Comment (QuentinBisson) [reply helm/dashboards/dashboards/kvm/kvm-machine-usage.json:1]: You are right  kept the wrong one

File: helm/dashboards/templates/configmap.yaml [hunk 3]
```diff
@@ -1,31 +1,62 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
+  annotations:
+    k8s-sidecar-target-directory: "/var/lib/grafana/dashboards/public/mixin"
```
[the remark this answers is not in the document]
Comment (QuentinBisson) [reply helm/dashboards/templates/configmap.yaml:5]: This annotation is going used by the k8s-sidecar used by the grafana app

[the remark this answers is not in the document]
Comment (QuentinBisson) [reply helm/dashboards/templates/configmap.yaml:5]: That's to try and see if we can have ererything under Public / Private and not need subfolders

[the remark this answers is not in the document]
Comment (TheoBrigitte) [reply helm/dashboards/templates/configmap.yaml:5]: I would not add it now, but add it later when we know the sidecar dashboard import works.

[the remark this answers is not in the document]
Comment (QuentinBisson) [reply helm/dashboards/templates/configmap.yaml:5]: Yes sure but I needed a way to test if without merging the big dashboard PR ;D

[the remark this answers is not in the document]
Comment (QuentinBisson) [reply helm/dashboards/templates/configmap.yaml:5]: I'll remove and create a new branch

[the remark this answers is not in the document]
Comment (QuentinBisson) [reply helm/dashboards/templates/configmap.yaml]: Done :)

**Comment by QuentinBisson** [comment]
@TheoBrigitte can you take another look ?

**Pull request closed without merging**